elementary/config/config.py

```diff
@@ -91,3 +93,7 @@ def _validate_elementary_profile(self):
                 raise NoElementaryProfileError
         except FileNotFoundError:
             raise NoProfilesFileError(self.profiles_dir)
+
+    def validate_timezone(self):
+      if self.timezone and not tz.gettz(self.timezone):
+         raise InvalidArgumentsError('An invalid timezone was provided to Slack alerts.')
```

Comment:
Given that you change it to be generic and not necessarily coupled to Slack, let's change the message accordingly.
```suggestion
         raise InvalidArgumentsError('An invalid timezone was provided.')
```